### PR TITLE
Implemented test for frontend LoginPage and RegistrationPage

### LINKED ISSUE #48: Implement testing for login and registration page frontend

### PR DESCRIPTION
**Pull request by nasaku898**
resolves #48

### CODE REVIEW

File: bike-erp/src/test/RegistrationPage/RegistrationPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import RegistrationPage from '../../components/RegistrationPage/RegistrationPage'
+import { shallow, configure, mount } from 'enzyme'
+import store from '../../redux/store'
+import Adapter from 'enzyme-adapter-react-16';
+import renderer from 'react-test-renderer'
+
```
Comment (Manraj45) [inline bike-erp/src/test/RegistrationPage/RegistrationPage.test.tsx:7]: Add comments

**Review by Manraj45 — CHANGES_REQUESTED** [review]
Everything seems goods but you need to add some comments in your tests

File: bike-erp/src/test/LoginPage/LoginPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import LoginPage from '../../components/LoginPage/LoginPage'
+import { shallow, configure } from 'enzyme'
+import store from '../../redux/store'
+import Adapter from 'enzyme-adapter-react-16';
+import renderer from 'react-test-renderer'
+import { BrowserRouter, BrowserRouter as Router } from 'react-router-dom';
+configure({ adapter: new Adapter() });
+
+const setUp = () => {
```
[the remark this answers is not in the document]
Comment (nasaku898) [reply bike-erp/src/test/LoginPage/LoginPage.test.tsx:11]: Great point, will add them!

File: bike-erp/src/test/RegistrationPage/RegistrationPage.test.tsx (hunk 1 above)
Comment (nasaku898) [reply bike-erp/src/test/RegistrationPage/RegistrationPage.test.tsx:7]: Done!

**Review by Manraj45 — APPROVED** [review]
Good Job

**Review by Mnx458 — APPROVED** [review]

**Pull request merged**